driver/compmisc.ml

```diff
@@ -13,6 +13,10 @@
 (*                                                                        *)
 (**************************************************************************)
 
+let auto_include =
+  if !Clflags.no_std_include then fun _ _ -> raise Not_found
+  else Load_path.auto_include_otherlibs Location.auto_include_alert
```

Comment:
This reads the value of `Clflags.no_std_include` when `Compmisc` is initialized which happens before CLI argument are parsed.

Reply:
Oops (again) - done!